understanding/20/focus-order.html

```diff
@@ -54,6 +54,28 @@ <h2>Intent of Focus Order</h2>
          Web page. Care should be taken so that the focus order makes sense to both of these
          sets of users and does not appear to either of them to jump around randomly. 
       </p>
+
+      <p>Focus order also does not need to follow the visual presentation of the web page,
+         as long as the order in which elements receive focus is logical, and the hierarchy and relationship
+         of content implied by the visual presentation is preserved. For instance, in a two-column
+         layout, visually the assumption in western left-to-right systems may be that elements in the
+         left-hand column would receive focus first. However, if meaning and operation are not affected,
+         it is not a failure if elements in the right-hand column receive focus first.
+         Again, as a best practice, care should be taken to make sure the focus order does not
+         appear random and illogical.
+      </p>
+
+      <div class="note">
+         <p>It is not a failure of this success criterion if static/non-interactive content
+         receives focus. Making an element that does not act as an actionable user interface component
+         focusable, or programmatically moving focus to such an element, does not necessarily constitute
+         a failure of this success criterion. This situation is only failures if these additional focus stops
+         significantly impede the operation of the content, or create illogical focus orders - for example, a control
+         appearing to receive focus multiple times due to the use of nested focusable elements.</p>
+         <pre><code>&lt;div tabindex="0"&gt;&lt;button&gt;...&lt;/button&gt;&lt;/div&gt;</code></pre>
+         <p>As a best practice, authors should avoid excessive use of focusable elements which cannot be operated
```

Comment:
We get dinged on use of "should", how about:
"As a best practice avoid excessive use of..."

Reply:
I would remove the word "excessive" here. People _should_ avoid use of focus on inoperable elements. Having to do so implies, to me, some failure in the design. If we're saying it's a best practice, I don't think we need to quantify it.

Reply:
I agree with removing the word <q>excessive</q>.  I am glad not have tab stop on `P` be a failure, but agree that developers should avoid use of focus on inoperable elements.

Reply:
Tweaked